deb/openmediavault/var/www/openmediavault/js/omv/util/i18n.js

```diff
@@ -65,6 +65,19 @@ OMV.util.i18n = function() {
 			Ext.util.Cookies.set(cookieName, locale, expires);
 		},
 
+		/**
+		 * Test if locale is RTL
+		 */
+		isRTL: function() {
+			locale = this.getLocale();
+			return ['ar_SA'].includes(locale);
+		},
+			locale = this.getLocale();
```

Comment:
You need to remove the lines 
```
		locale = this.getLocale();
			if(locale === 'ar_SA')
				return true
			return false;
		},
```
otherwise this will create a runtime error.

Reply:
removed! 

Thanks votdev